src/dsep.jl

```diff
@@ -6,14 +6,18 @@ Check  whether `u` and `v` are d-separated given set `s`.
 Algorithm: unrolled https://arxiv.org/abs/1304.1505
 
 """
-function dsep(g::AbstractGraph, u::Integer, v::Integer, S; verbose = false)
+function dsep(g::AbstractGraph, u::Integer, v::Integer, S; sel=[], verbose = false)
```

Comment:
I dislike this change, for example now it makes a copy where no copy is needed. Can you adapt the calling function instead?

Reply:
fixed.